cpp/ps0003/pooo3-soln.cpp

```diff
@@ -1,17 +1,23 @@
-# include <iostream>
+#include <iostream>
+
 using namespace std;
+
 int main(){
-int number;
-cout<<"Enter your number:";
-cin>> number;
-if (number%2==0)
-{
-cout<<"the number "<<number<<" is even";
+  // prompting user for number 2/2
+  int number;
+  cout << "Enter your number:";
+  cin >> number;
+  
+  // checking if the number is odd or even 3/3
+  if (number % 2 == 0)
+  {
+    cout << "the number " << number << " is even";
+  }
+  else 
+  {
+    cout << "the number " << number << " is odd";
+  }
+  
+  return 0;
 }
-else 
-{
-  cout<<"the number "<<number<< "is odd" ;
-}
-
-return 0;
-}
\ No newline at end of file
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code